src/components/IterableEditComponent.tsx

```diff
@@ -317,15 +323,16 @@ export class IterableEditComponent extends React.Component<ImageUploadProps | It
                         }
 
                         if (this.props.field.iterabletype && this.props.field.iterabletype.type === "typeahead") {
-                            return <div key={"iterable" + this.props.field.iterabletype.type + index}
+                            return <div key={"iterable" + this.props.field.iterabletype.type + index + this.state.model[index]}
                                 style={this.props.field.iterabletype.displayChildren === "inline" ? {
```

Comment:
Added defined key for iterable.
 As List will not uniquely defined by index in key. It causing error with addAtIndex button.